roles/softserve/templates/softserve.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=backend server for softserve
+After=network.target
+
+[Service]
+Type=simple
+PIDFile=/run/softserve/softserve.pid
+WorkingDirectory=/softserve/code/
+ExecStart=/softserve/env/bin/gunicorn --pid /run/softserve/softserve.pid -w 4 softserve:app
+User={{ user }}
+
+PrivateTmp=true
```

Comment:
Nice to see hardening, could we get some more ? ( like PrivateDevices= , ProtectSystem= ProtectHome= , NoNewPrivileges= ). In a ideal world, i would also ask for a selinux module, but it might be overkill (and a bit complicated)